### ISSUE TITLE
Domains and labor categories

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#56]
As a buyer or seller, I need to orient myself with the specifics of the domains and labor categories, so that I can determine if I want to solicit work through this IDIQ (buyer) or offer my services to the government (seller).

- [Content](https://docs.google.com/document/d/12BqJRXAbaBr-qHOIOoSD4d_QPOOSiCL7wndqwcGhD38/edit#heading=h.iygv7gu3pcbe)
- [Design](https://www.figma.com/proto/lWfZDygV7WbvGlea2PIrpa/OASIS-Playground?page-id=&type=design&node-id=2397-16547&viewport=-1820%2C678%2C0.13&t=nfMGgufk1vKGLB2P-1&scaling=min-zoom&starting-point-node-id=2389%3A11651)

**Status**

- [x]  Content created
- [x]  Content reviewed by SD
- [x]  Content reviewed by SME
- [x]  Design finalized
- [x]  Content finalized
- [x]  Page developed

**Comment by PowderD** [comment]
@ekidenda do we need to have text for the The enterprise solutions domain?  I think we should still have the description of enterprise solutions and then add the text currently there stating it's only for UNR.  

This is from the OASIS+ Intro deck

"Large, complex, integrated professional service requirements often spanning multiple disciplines and/or locations and requiring many different types of labor and expertise."

**Comment by ekidenda** [comment]
Yes. @jfredrickson5 Let's use the language below for enterprise solutions domain. Thanks:

" The enterprise solutions domain will only be available for the Unrestricted contract family. The services on this domain include large, complex, integrated professional service requirements often spanning multiple disciplines and/or locations, and requiring many different types of labor and expertise. "

**Comment by klohman** [comment]
@ekidenda The content update for Enterprise Solutions has been made. Moving to Done.
